week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,29 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+// exercise 1
```

Comment:
Try console logging the result of the percentage function and then use the percentage function within the percentageFinal function